### PR TITLE
Fix 4544: Update all occurrences of open-event-orga-server

### LINKED ISSUE #4544: Update installation/deployment instructions.
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-orga-server

**Current behavior:**
Currently in all the installation/deployment instructions there are many instances where `open-event-orga-server` still persists which can create confusion while following the instructions.

**Expected behavior:**
`open-event-orga-server` needs to be replaced with `open-event-server`at many instances in installation/deployment docs.

**Steps to reproduce:**
Check out the documentation of different installations/deployments.
https://github.com/fossasia/open-event-server/tree/development/docs/installation and https://github.com/fossasia/open-event-server/tree/development/docs

### PR DESCRIPTION
**Pull request by raju249**
Fixes #4544 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

Fixes #4544 All the occurrences of `open-event-orga-server` are replaced with `open-event-server`.
Gitter, Travis and Codecov URLs are kept as it is.
#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by raju249** [comment]
Yes @anshumanv 
Sorry I missed it.
Commit on its way.

**Comment by invinciblycool** [comment]
@anshumanv
Included already in https://github.com/fossasia/open-event-server/pull/4540

**Comment by anshumanv** [comment]
Ah i missed it, thanks for the follow up @invinciblycool :smile:

**Comment by raju249** [comment]
No need for the commit then I guess 
@anshumanv ?

**Comment by anshumanv** [comment]
No need @raju249, #4540 will require changes and can be merged. 
Good work here. :+1:

**Comment by raju249** [comment]
Thanks @anshumanv

File: kubernetes/travis/deploy-nextgen.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 export NEXTGEN_DEPLOY_BRANCH=${NEXTGEN_DEPLOY_BRANCH:-nextgen}
```
Comment (poush) [inline kubernetes/travis/deploy-nextgen.sh]: Can you please change it to "development". Make sure there is no "nextgen" in this file.

Comment (raju249) [reply kubernetes/travis/deploy-nextgen.sh]: Should I remove all occurrences of nextgen ?
Or just this file ?

Comment (poush) [reply kubernetes/travis/deploy-nextgen.sh]: nextgen in readme file is taken care in an existing PR. You can update any other file.

Comment (raju249) [reply kubernetes/travis/deploy-nextgen.sh]: Ok, just updated the file that you said. Checking for others

Comment (raju249) [reply kubernetes/travis/deploy-nextgen.sh]: What about the file name ?

**Comment by raju249** [comment]
@poush Integrated the suggestions :smiley:

File: kubernetes/yamls/api-server/deployment.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
         emptyDir: {}
       containers:
       - name: api-server
-        image: eventyay/nextgen-api-server:latest
```
Comment (poush) [inline kubernetes/yamls/api-server/deployment.yml]: It is still nextgen-api-server at docker https://hub.docker.com/r/eventyay/nextgen-api-server/

@mariobehling @niranjan94 Can you please look into it.

Comment (raju249) [reply kubernetes/yamls/api-server/deployment.yml]: Should I change it back to `nextgen` ? @poush

**Comment by raju249** [comment]
@shubham-padia Suggestions acknowledged!

**Comment by shubham-padia** [comment]
Thanks @raju249

**Pull request merged**